server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -1084,16 +1062,14 @@ private List<File> linkEngine(List<String> symbols, Map<String, Object> manifest
         for (String template : commands) {
             String command = templateExecutor.execute(template, context);
             if (resolvedPods != null) {
-                command = command.replace("-miphoneos-version-min=9.0", "-miphoneos-version-min=" + resolvedPods.platformVersion);
+                command = command.replace("-miphoneos-version-min=9.0", "-miphoneos-version-min=" + resolvedPods.platformMinVersion);
```

Comment:
Do you have a migration plan for this?

E.g. when we change this line in the `build.yml`, the server will not detect this.

Reply:
Removed in favour of osMinVersion in the context